seed_identity_store/settings.py

```diff
@@ -150,7 +162,7 @@
 CELERY_CREATE_MISSING_QUEUES = True
 
 CELERY_TASK_SERIALIZER = 'json'
-CELERY_RESULT_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'pickle'
```

Comment:
Eep! Does something need this? Passing pickles for celery results makes debugging celery results rather tricky (one needs to be able to unpickle them in an exact copy of the production setup, possibly with access to the production db, and possibly triggering arbitrary code).